util.go

```diff
@@ -67,8 +62,22 @@ func getService(c *cli.Context) (*route53.Route53, error) {
 	config, err := getConfig(c)
 	if err != nil {
 		return nil, err
-	}
-	return route53.New(session.New(), config), nil
+  }
+  profile := c.String("profile")
+  if profile != "" {
+    sess := session.Must(session.NewSessionWithOptions(session.Options{
+       Config: *config,
+       SharedConfigState: session.SharedConfigEnable,
+       Profile: profile,
+    }))
+    return route53.New(sess), nil
+  } else {
+    sess := session.Must(session.NewSessionWithOptions(session.Options{
```

Comment:
This whole block of code is duplicated in both branches of the condition - when in fact it should just set `.Profile` on `session.Options` when `profile != ""`